bluesky_queueserver/server/tests/test_http_server.py

```diff
@@ -104,45 +106,56 @@ def test_http_server_close_environment_handler(re_manager, aiohttp_server):  # n
     resp1 = _request_to_json('post', '/create_environment')
     assert resp1 == {'success': True, 'msg': ''}
 
+    ttime.sleep(1)  # TODO: API needed to test if environment initialization is finished. Use delay for now.
+
     resp2 = _request_to_json('post', '/close_environment')
     assert resp2 == {'success': True, 'msg': ''}
 
+    ttime.sleep(3)  # TODO: API needed to test if environment is closed. Use delay for now.
+
     resp3 = _request_to_json('post', '/close_environment')
     assert resp3 == {'success': False, 'msg': 'Environment does not exist.'}
 
 
 def test_http_server_process_queue_handler(re_manager, aiohttp_server, add_plans_to_queue):  # noqa F811
     resp1 = _request_to_json('post', '/process_queue')
-    assert resp1 == {'success': False, 'msg': 'Environment does not exist. Can not start the task.'}
+    assert resp1 == {'success': False, 'msg': 'RE Worker environment does not exist.'}
 
     resp2 = _request_to_json('post', '/create_environment')
     assert resp2 == {'success': True, 'msg': ''}
     resp2a = _request_to_json('get', '/queue_view')
     assert len(resp2a['queue']) == 3
 
+    ttime.sleep(1)  # TODO: API needed to test if environment initialization is finished. Use delay for now.
+
     resp3 = _request_to_json('post', '/process_queue')
     assert resp3 == {'success': True, 'msg': ''}
-    resp3a = _request_to_json('get', '/queue_view')
-    assert len(resp3a['queue']) == 2
+    # resp3a = _request_to_json('get', '/queue_view')
+    # assert len(resp3a['queue']) == 2
 
-    resp4 = _request_to_json('post', '/process_queue')
-    assert resp4 == {'success': True, 'msg': ''}
-    resp4a = _request_to_json('get', '/queue_view')
-    assert len(resp4a['queue']) == 1
+    # resp4 = _request_to_json('post', '/process_queue')
+    # assert resp4 == {'success': True, 'msg': ''}
+    # resp4a = _request_to_json('get', '/queue_view')
+    # assert len(resp4a['queue']) == 1
+
+    # resp5 = _request_to_json('post', '/process_queue')
+    # assert resp5 == {'success': True, 'msg': ''}
```

Comment:
Ah, I fixed it in one place via https://github.com/bluesky/bluesky-queueserver/pull/42/commits/c9dce7c8dfcce8d27bd1a0b1d4c9b91ca62cbdea, but not here. Thanks for commenting it. I think it can even be removed as it was my wrong understanding of the workflow.

Reply:
I cleaned it up via https://github.com/bluesky/bluesky-queueserver/pull/43/commits/164ae805fea0bf4cc3b030fb4ffed639dc81ad8e.